tests/Stub/Entity/Operation.php

```diff
@@ -213,12 +213,13 @@ public function setWithdrawnAmount($withdrawnAmount)
     {
         $this->withdrawnAmount = $withdrawnAmount;
     }
-    function getOriginAmount()
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=php%3AS1131)